portfolio/src/main/webapp/index.html

```diff
@@ -161,6 +163,47 @@ <h2>~ hiking ~</h2>
         <div id="hiking-map"></div>
       </div>
     </section>
+    <section id="personality">
+      <div>
+        <h2>~ personality type chart ~</h2>
+        <p class="section-descr">
+          What's your Myers-Briggs Type Indicator personality type? There are four dimensions: 
+        </p>
+        <p>
+          <ul>
+            <li>Where you focus your attention - extraversion (E) or introversion(I)</li>
+            <li>The way you take in information - sensing (S) or intuition (N)</li>
+            <li>How you make decisions - thinking (T) or feeling (F)</li>
+            <li>How you deal with the world - judging (J) or perceiving (P)</li>
+          </ul>
+        </p>
+        <p class="section-descr">
+          Take the test <a href="https://www.16personalities.com/free-personality-test">here</a>!
```

Comment:
I remember there being so many questions to this test, I can't remember what type I am